src/SSEConnection.js

```diff
@@ -0,0 +1,54 @@
+import AbstractConnection from "./AbstractConnection";
+import ConnectionEvent from "./ConnectionEvent";
+
+/**
+ * Server Sent Event Connections
+ */
+export default class SSEConnection extends AbstractConnection {
+  constructor(url, options = {}) {
+    super(url);
+    this.options = options;
+    this.source = null;
+    this.data = [];
```

Comment:
can you please add a `getter` for this? the Idea of these connections is to have them as restricted as possible, right now this is `public` and coudl be changed by everyone - most probably this one should be `private`, but at least `protected` 🤔 
(I'm aware that this isnt possible in JavaScript, but if you have a look at `XHRConnection` we declared them as protected for reference, once we switch to typescript)

Reply:
ah okay, thank you for explaining the idea of restriction here, I wasn't aware of it.